comp/networkpath/npcollector/npcollectorimpl/npcollector.go

```diff
@@ -28,13 +29,15 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/networkpath/traceroute"
 	"github.com/DataDog/datadog-agent/pkg/networkpath/traceroute/config"
 	"github.com/DataDog/datadog-agent/pkg/process/statsd"
+	"github.com/DataDog/datadog-agent/pkg/util/cloudproviders"
 )
 
 const (
 	networkPathCollectorMetricPrefix    = "datadog.network_path.collector."
 	reverseDNSLookupMetricPrefix        = networkPathCollectorMetricPrefix + "reverse_dns_lookup."
 	reverseDNSLookupFailuresMetricName  = reverseDNSLookupMetricPrefix + "failures"
 	reverseDNSLookupSuccessesMetricName = reverseDNSLookupMetricPrefix + "successes"
+	netpathConnsSkippedMetricName       = networkPathCollectorMetricPrefix + "conns_skipped"
```

Comment:
```suggestion
	netpathConnsSkippedMetricName       = networkPathCollectorMetricPrefix + "schedule.conns_received"
```

To be consistent with:
```
datadog.network_path.collector.schedule.conns_received
```

Reply:
Updated with this metric